styles/duckduckgo/catppuccin.user.css

```diff
@@ -510,11 +510,14 @@
       color: @crust !important;
     }
 
-    .js-region-filter-switch,
+ .js-region-filter-switch,
     .frm__switch__label {
-      background-color: @accent-color !important;
+      background-color: @overlay0;
       border-color: @accent-color !important;
     }
+    .switch.is-on {
+      background-color: @accent-color;
```

Comment:
Seems like `!important` is needed here so that the color really is being applied (now the color (some blue) from duckduckgo isn't changed when the switch is active)